app/src/main/java/com/example/concordia_campus_guide/Activities/PathsViewModel.java

```diff
@@ -16,40 +16,40 @@ public Place getFrom() {
         return SelectingToFromState.getFrom();
     }
 
-    public Place getEntrance(Place place){
+    public Place getEntrance(Place place) {
         if (place instanceof RoomModel) {
             String campusCode = ((RoomModel) place).getCampus();
             String floorCode = ((RoomModel) place).getFloorCode();
-            String buildingCode = floorCode.substring(0, floorCode.indexOf("-")).toUpperCase();
+            String buildingCode = floorCode.substring(0, floorCode.indexOf('-')).toUpperCase();
             //For indoor direction, no need to return the actual room from the db as the room coordinates will not be used by the PathFinder
```

Comment:
Is this comment used a reminder for something?

Reply:
nope t's just an explanation why we're creating a new room